### ISSUE TITLE
The Immortal Soldier

### ISSUE THREAD

**Issue by Silvergnom** [jsettlers/settlers-remake#640]
Greetings,

I just had a game where a soldier in an enemy tower was invincible and killed 100 of mine with just one hit for each of them.

![theimmortal](https://user-images.githubusercontent.com/29957146/27997337-638eb550-64f6-11e7-959a-3fb740a9b65b.png)
![theimmortal2](https://user-images.githubusercontent.com/29957146/27997338-692fe5f6-64f6-11e7-9af9-24e9b8530ee3.png)

While he clearly killed my people, he hasnt performed his attack animation while doing it.
Just stood there and looked at mine.
Maybe it was Chuck Norris.

[2017-07-09_22-07-02_6_players_fun.zip](https://github.com/jsettlers/settlers-remake/files/1134042/2017-07-09_22-07-02_6_players_fun.zip)

Version: d09d53cc

**Comment by benhandschuh** [comment]
😂

**Comment by DerKrawallkeks** [comment]
I agree, it must have been Chuck Norris. There is no way to remove his abilities from the game.
